runbotics/runbotics-scheduler/src/scheduler-database/tags/tag.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn, Unique } from 'typeorm';
+import { Tenant } from '../tenant/tenant.entity';
+
+@Entity()
+@Unique(['name', 'tenantId'])
+export class Tag {
+    @PrimaryGeneratedColumn({ type: 'bigint' })
+    id: number;
+
+    @Column({ type: 'varchar', length: 20 })
+    name: string;
+
+    @Column({ name: 'tenant_id', type: 'uuid' })
+    tenantId: string;
+
+    @ManyToOne(() => Tenant, tenant => tenant.id, { nullable: false })
```

Comment:
why is `nullable: false` here?

Reply:
becuase once we implemented tenants no resource can be created without tenant related to this, the default is { nullable: true }